docs/config.md

```diff
@@ -135,6 +135,11 @@ version = 2
       # i.e pass host devices through to privileged containers.
       privileged_without_host_devices = false
 
+      # base_runtime_spec is a file path to a JSON file with the OCI spec that will be used as the base spec
+      # that all container's are created from.
+      # Use containerd's `ctr oci default-spec > /etc/containerd/cri-base.json` to output initial spec file.
+      base_runtime_spec = ""
```

Comment:
nit.. may want to mention a restart of containerd is required to load changes to the one default spec...
still running containers and restarted containers will still be using the original spec from which that container was created.

Reply:
Done